spec/lib/twingly/url_spec.rb

```diff
@@ -102,6 +114,16 @@ def valid_urls
       end
     end
 
+    context "IDNA 2003 vs IDNA 2008" do
```

Comment:
> considering we want to do things like #71. That I'm using normalized here is just me "cheating"... :)

I don't think it's cheating (as it seems like now, we need to `#normalize`to trigger this, if I am not mistaken) I believe it should be put under that description. Easier to find and it's what we do currently for the most parts. 

I think it's hard to reason about #71 at this stage as it makes everything more complex, let's take that discussion when we get there 😄